andela/knights-bn-backend
# #171186789 Fix ESLint and HoundCI Error Messages

#### What does this PR do?
- This PR enhances good code quality as required by ESLint and HoundCI

#### Description of Task to be completed?
- Setting the rules inside the eslintrc.json file correctly. 
- Deleting obsolete eslintrc.js

#### How should this be manually tested?

- HoundCI should not throw *no-def* and *'import' and 'export' may appear only with 'sourceType: module'*  errors

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#171186789](https://www.pivotaltracker.com/n/projects/2431452)

#### Screenshots (if appropriate)
N/A
